src/image_opencv.cpp

```diff
@@ -591,25 +591,49 @@ extern "C" cap_cv* get_capture_video_stream(const char *path) {
 
 extern "C" cap_cv* get_capture_webcam(int index)
 {
-    cv::VideoCapture* cap = NULL;
-    try {
+#ifdef REALSENSE2
+	printf("[RDV] Use Realsense2 Camea\n") ;
+
+	CRealsense *cap = NULL;
+
+	try 
+	{
+		cap = new CRealsense() ;
+	}	
+#else
+	printf("[RDV] Use Opencv Video Capture\n") ;
+
+	cv::VideoCapture* cap = NULL;
+    try 
+	{
```

Comment:
@juhong-rdv Hi,

Is there any advantage to using `cap = new CRealsense() ;` over `cap = new cv::VideoCapture(index);` for Intel-Realsense RGB?

Is it better to integrate Intel-Realsense RGBD to the https://github.com/AlexeyAB/darknet/blob/master/src/yolo_console_dll.cpp as it is done for ZED-Camera, so it can show Distance to the objects too?